### ISSUE TITLE
Compiler error with IAR

### ISSUE THREAD

**Issue by ghseb** [PelionIoT/mbed-cloud-client#95]
The line 

https://github.com/PelionIoT/mbed-cloud-client/blob/105a1846277c80e5eb15ad74de7e4d7b15e57c26/mbed-client/source/m2minterfaceimpl.cpp#L978

produces a compiler error on IAR:
```
m2minterfaceimpl.cpp",978  Error[Pe2486]: user-defined literal operator not found
```

It could be fixed by adding spaces before and after `PRId32` in order to be not evaluated as user defined literal operator.

**Comment by ghseb** [comment]
Ok, thx. Closing the issue.

**Comment by teetak01** [comment]
This was fixed in 4.11.1 patch release.
